resources/chefdk/msi/assets/start-chefdk.ps1

```diff
@@ -0,0 +1,22 @@
+$conemulocation = "$env:programfiles\ConEmu\Conemu64.exe"
+$chefdkinit = 'chef shell-init powershell | iex; Import-Module chef; $env:CHEFDK_ENV_FIX = 1'
+$chefdkgreeting = "echo 'PowerShell $($PSVersionTable.psversion.tostring()) ($([System.Environment]::OSVersion.VersionString))';write-host -foregroundcolor darkyellow 'Ohai, welcome to Chef-DK!`n'"
```

Comment:
We probably need to do said localization, but I'm not sure how to get the .ps1 to access the localized strings in the wxs file.